### DISCUSSION TITLE
NEVER power the RAD from both external and internal - so of course I did ...

### DISCUSSION THREAD

**Discussion by mannalex** [frntc/RAD#34] [Q&A]
I accidentally powered the C64 on with having the Jumper in and the Raspi powered as well. 
Now I have shaky characters and graphics but everything else seems to work alright. Any idea what I could do to find out what's broken now and how to fix it?
Thanks for any help!

Here's a little video snipped of the issue.
https://github.com/frntc/RAD/assets/18057397/759a2d72-bff6-4110-b626-c685f92df9d8

And the diagnostics results:
![IMG_6141](https://github.com/frntc/RAD/assets/18057397/fdb8609b-efe8-4dd1-8cc7-56111381d0e6)

**Comment by frntc** [comment]
Did you have the diagnostic harness attached to the C64? It shows that CIAs are defective. One of them controls the VIC bank, which could, I think, be a reason for weird characters. Do you have another C64 in which you can try the suspicious CIAs? (not vice versa!)

**Comment by mannalex** [comment]
Thanks for the response.
I started a thread in the forum64 (https://www.forum64.de/index.php?thread/141202-c64-mit-rad-expansion-unit-%C3%BCberladen/) - so I'm closing the discussion here.
